tests/sagemaker/test_model_export.py

```diff
@@ -1,71 +0,0 @@
-from __future__ import print_function
```

Comment:
This suite consists of a single test that attempts to score a pyfunc-wrapped scikit-learn model by deploying it inside a Docker container. A similar test has been implemented in the `tests.pyfunc.test_model_export` suite; it is called `test_sagemaker_docker_model_scoring_with_default_conda_env`. Therefore, we propose removing this test suite.